src/ovirtapi/transport.js

```diff
@@ -57,27 +57,32 @@ type GetRequestType = { url: string, custHeaders?: Object}
 type InputRequestType = { url: string, input: string, contentType?: string }
 type DeleteRequestType = { url: string, custHeaders?: Object }
 
+let getCounter = 0
+const logHeaders = (headers) => JSON.stringify({ ...headers, 'Authorization': '*****' })
+
 function httpGet ({ url, custHeaders = {} }: GetRequestType): Promise<Object> {
-  logger.log(`_httpGet start: url="${url}"`)
+  const myCounter = getCounter++
```

Comment:
I don't see any usage of this, so think it can be removed.

Reply:
It's being used by the logger function lower down.  I added it so the request and responses can be matched in the log when fetches are being run in parallel.

Reply:
Oh, yes, I missed it, thank you.